diff --git a/common/cmake/embree-config-linux.cmake b/common/cmake/embree-config-linux.cmake
index b53cf98..39e536b 100644
--- a/common/cmake/embree-config-linux.cmake
+++ b/common/cmake/embree-config-linux.cmake
@@ -14,7 +14,11 @@
 ## limitations under the License.                                           ##
 ## ======================================================================== ##
 
+IF(CMAKE_LIBRARY_ARCHITECTURE)
+SET(EMBREE_INCLUDE_DIRS ${CMAKE_CURRENT_LIST_DIR}/../../../../@CMAKE_INSTALL_INCLUDEDIR@)
+ELSE(CMAKE_LIBRARY_ARCHITECTURE)
 SET(EMBREE_INCLUDE_DIRS ${CMAKE_CURRENT_LIST_DIR}/../../../@CMAKE_INSTALL_INCLUDEDIR@)
+ENDIF(CMAKE_LIBRARY_ARCHITECTURE)
 SET(EMBREE_LIBRARY ${CMAKE_CURRENT_LIST_DIR}/../../libembree.so.@EMBREE_CONFIG_VERSION@)
 SET(EMBREE_LIBRARY_XEONPHI ${CMAKE_CURRENT_LIST_DIR}/../../libembree_xeonphi.so.@EMBREE_CONFIG_VERSION@)
 
